website/docs/reference/deploy/environment-import-export.mdx

```diff
@@ -0,0 +1,110 @@
+The exported and imported context fields have the same name but the target environment doesn't support some of the source environment context field values. If the context fields don't exist in the target environment they will be created.
+
+#### Segments
+
+The target Unleash instance doesn't have a matching segment. The actual constraints in the segment can be different. The matching is done by the segment name.
+
+#### Custom strategies
+
+The exported strategies include custom strategies that don't exist in the target Unleash instance. The matching is done by the strategy name.
+
+#### Existing features
+
+You're trying to import a feature that already exists in a different project on the target instance.
+
+#### Pending change request
+
+The target project and environment have a pending change request (applies to enterprise customers with change requests enabled for a given project and environment).
+
+### Import warnings
+
+Import validation will warn you when:
+* the features you are trying to import are already archived in the target instance in any project. Those features won't be imported again.
+* the strategies you are trying to import are custom and may have different [configuration parameters](../custom-activation-strategies.md#parameters). Those strategies will be imported anyway.
+
+### Required permissions
+
+To import features, you will **always** require the **update feature toggles** permission.
+Additionally, depending on how the actions your import job would trigger, you may also require any of the following permissions:
+* **Create feature toggles**: When the import would create new features
+* **Update tag types**: when the import would create new tag types
+* **Create context fields**: when the import would create new context fields
+* **Create activation strategies**: when the import would add (or is this "create custom activation strategies"? custom or standard?) activation strategies to a feature and change requests are disabled
+* **Delete activation strategies**: when import would remove existing activation strategies from a feature and change requests are disabled
+* **Update variants**: when the import would update variants and change requests are disabled
+
+
+### Import and change requests
+
+When the target project and environment have change requests enabled the import will not be applied instantly. Instead, it will
+create a change request, send it to review and await for the approval.
+
+If change requests are enabled, any permissions for  **Create activation strategies**, **Delete activation strategies** and **Update variants** are not required.
+
+## Environment Import & Export vs Old Import & Export
+
+The new Environment Import & Export differs from the old [Import & Export](import-export.md).
+The most important difference is a strong focus on a single project and environment in the target environment.
+Also, the new Import & Export has a UI and extra safety mechanisms preventing corrupted data imports.
```

Comment:
I don't have the time right now, but I'll get back to this later. I think it could do with some fleshing out.

Reply:
Maybe something like this?

```suggestion
## Environment import/export vs the [instance import/export API](import-export.md)

Environment import/export has some similarities to the [instance import/export API](import-export.md), but they serve different purposes. 

The instance import/export API was designed to export all feature toggles (optionally with strategies and projects) from one Unleash instance to another. When it was developed, Unleash had much fewer features than it does now. As such, the API lacks support for some of the more recent features in Unleash.

On the other hand, the environment import/export feature was designed to export a selection of features based on search criteria. It can only export data from a single environment and only import it to a single environment. It also only supports importing into a single project (although it can export features from multiple projects).

Further, the environment import/export comes with a much more stringent validation and will attempt to stop any corrupted data imports.
```